src/main/java/io/vertx/rabbitmq/RabbitMQConfirmListener.java

```diff
@@ -0,0 +1,78 @@
+package io.vertx.rabbitmq;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.Handler;
+import io.vertx.core.streams.ReadStream;
+
+/**
+ *
+ * @author jtalbut
+ */
+@VertxGen
+public interface RabbitMQConfirmListener extends ReadStream<RabbitMQConfirmation> {
```

Comment:
I don't think we need such type as it does not bring anything specific than the `ReadStream` does not provide already beside isPaused()